repository/confluent-schema-registry/docs/v1.0/security.md

```diff
@@ -0,0 +1,103 @@
+
+## Encryption
+
+By default, KUDO Kafka brokers use the plaintext protocol for its inter-broker communication. It is recommended to enable the TLS encryption, to secure the communication between brokers. The Confluent Schema Registry can be configured to connect with Kafka brokers over a secure connection with an appropriate encryption mechanism.
+
+### Enabling TLS encryption
+
+To connect Confluent Schema Registry with Kafka Brokers configured with TLS, get the TLS certificate and key that is used for Kafka TLS encryptions to create a new Kubernetes TLS secret for Confluent Schema Registry. 
+
+Here, we are assuming that the TLS Certificate file name is `tls.cert` and key name is `tls.key` that is used for Kafka TLS encryptions. We create a new Kubernetes TLS secret for Confluent Schema Registry as:
+
+```
+$ kubectl create secret tls confluent-schema-registry-tls --cert=tls.crt --key=tls.key
+```
+
+:warning: Make sure to create the certificate in the same namespace where the Confluent Schema Registry is being installed.
+
+Now, to connect with Kafka Brokers configured with TLS:
+- Provide Fully Qualified Domain Name (FQDN) of each Kafka Broker(s) as `bootstrap.servers` parameter to connect with Kafka successfully. Eg. `KAFKA_BOOTSTRAP_SERVERS="SSL://kafka-kafka-0.kafka-svc.default.svc.cluster.local:9095,SSL://kafka-kafka-1.kafka-svc.default.svc.cluster.local:9095,SSL://kafka-kafka-2.kafka-svc.default.svc.cluster.local:9095"`.
+- Make the `TRANSPORT_ENCRYPTION_ENABLED` parameter value as `true`.
+- Supply the secret created before i.e. `confluent-schema-registry-tls` to be passed as parameter `TLS_SECRET_NAME`.
+
+```
+$ kubectl kudo install confluent-schema-registry \
+    --instance=schema-registry  \
+    -p KAFKA_BOOTSTRAP_SERVERS="SSL://kafka-kafka-0.kafka-svc.default.svc.cluster.local:9095,SSL://kafka-kafka-1.kafka-svc.default.svc.cluster.local:9095,SSL://kafka-kafka-2.kafka-svc.default.svc.cluster.local:9095" \
+    -p TRANSPORT_ENCRYPTION_ENABLED=true \
+    -p TLS_SECRET_NAME=confluent-schema-registry-tls
+```
+
+## Authentication
+
+Confluent Schema Registry service supports connecting with Kafka broker's configured with TLS authentication mechanism. TLS authentication requires TLS encryption to be enabled. 
+
+The configuration of TLS authentication while connecting with Kafka brokers is a superset of the required Kafka Broker's configurations that enables TLS encryption and also TLS authentication. Its not possible to enable only the TLS authentication without enabling the TLS encryption.
```

Comment:
This part is already in kafka's docs linked here https://github.com/kudobuilder/operators/pull/208/files#diff-92a4686d9d9fe796fdc4b8141e6d0794R42
KUDO SR doesn't  have any extra configuration to enable tls authentication

Reply:
Okay, will remove that.